New content for section: WIS2 Architecture.

6a6d74 opened the pull request:
Written to provide a high-level overview of things important to WIS2: roles and components.

References to the following resources are removed:
* sections/data-and-metadata-flows.adoc
* sections/real-time-data-exchange.adoc
* sections/discovery-metadata.adoc

I think these are no longer needed given the revamp of the Guide.

efucile reviewed (commented):
- where you say "WIS2 Nodes, GISCs and Global Services are types of WIS Centre". In the Manual we say that NC, DCPC and GISC are WIS Centres. NC, DCPC. I copy here the relevant part of the Manual
1.2.2 	NCs and DCPCs are responsible for publishing data and discovery metadata using a component referred to as a WIS node.
1.2.3 	GISCs are responsible for supporting WIS Centres in their Area of Responsibility (AoR) and ensuring the effective operation of WIS.
1.2.4 	GISCs may operate one or more global services that collectively ensure discovery of and access to data within all Regions.   
- where you say "a WIS Centre may operate one or more Global Services" this is not fully in line with 1.2.4 where we say GISCs ...We probably need to change the Manual.
- "Global Discovery Catalogue: enables users (humans and software agents)" are we suggesting that GDC has to provide a portal for humans to discover the data? I thought that the API is the core component of GDC and a portal for humans can be built on top of it. We could say that GDC may provide a  UI.
- as a general comment. In the Manual we say at the beginning of Vol. 2 that for WIS we mean WIS 2.0. We should do the same in the Guide.

efucile reviewed (commented):
from the Manual

3.7.5.6 	A Global Cache shall retain a copy of core data for a duration compatible with the real-time or near real-time schedule of the data and not less than 24-hours.

6a6d74 commented:
> "WIS2 Nodes, GISCs and Global Services are types of WIS Centre" ...

Updated to better reflect what the manual says. In (new) line 11 I say that a WIS Centre may operate one or more global services. Still true - but a little bit looser terminology than the manual. CA-ECCC isn't a GISC :)

> "Global Discovery Catalogue: enables users (humans and software agents)" ...

Removed the caveat about "humans and software agents" to avoid the confusion.

> 3.7.5.6 A Global Cache shall retain a copy of core data for a duration...

Amended the text here to use what's said in the manual.

6a6d74 commented:
> as a general comment. In the Manual we say at the beginning of Vol. 2 that for WIS we mean WIS 2.0. We should do the same in the Guide.

New issue raised (#68) for this. @efucile / @hhaddouch for action.

6a6d74 commented:
@tomkralidis - updates complete (see 33bc50ceda93f5d4184c27286af98a4f603fc297). Ready for further review and (hopefully!) merge.
